app/presenters/checkbox_question_presenter.rb

```diff
@@ -29,9 +29,13 @@ def checkboxes
     end
   end
 
+private
+
   def checked?(value)
-    return false if response_for_current_question.blank?
+    response = current_response
```

Comment:
So `current_response` comes from State, but is only set in StateResolver? is that correct?

Reply:
Can this commit be broken up so I can see which changes are related. For example, a lot of scrolling is required to see the affected checkbox views and tests.

Reply:
Also, when was `response_for_current_question` removed? Was it removed too early in the commit history?

Reply:
Yes `current_response` is a property set in StateResolver. It originates from params.

`response_for_current_question` was removed in https://github.com/alphagov/smart-answers/pull/5337/commits/8e1556202c68cc338fb37f03abc26d91c1f268c8 - I opted not to port it to remain working as part of that commit as that'd mean re-thinking it only to remove it.

I've broken the commits up now, they should be an easier read.